src/main_app/jobs_workers/public_jobs_workers/copy_svg_langs_per_file/steps/inject_one_file.py

```
@@ -0,0 +1,95 @@
+
+    if new_languages == 0 and stats.get("updated_translations", 0) == 0:
+        return InjectResult(result=None, msg="No changes")
+
+    try:
+        tree.write(str(output_file), encoding="utf-8", xml_declaration=True, pretty_print=True)  # type: ignore
+        return InjectResult(
+            result=True,
+            msg=f"Injected {new_languages} languages",
+            new_languages=new_languages,
+        )
+    except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)